content/en/docs/concepts/configuration/secret.md

```diff
@@ -143,7 +143,7 @@ method creates a new `Secret` object with the edited data.
 
 Depending on how you created the Secret, as well as how the Secret is used in
 your Pods, updates to existing `Secret` objects are propagated automatically to
-Pods that use the data. For more information, refer to [Mounted Secrets are updated automatically](#mounted-secrets-are-updated-automatically).
+Pods that use the data. For more information, refer to [Using Secrets as files from a Pod](#using-secrets-as-files-from-a-pod).
```

Comment:
This is wrong. We don't mean to change the referenced text. All you need is to point to the right link.

Reply:
Hmm, I followed what sftim said In https://github.com/kubernetes/website/pull/40103 so I thought u had to change it to point to a different topic

Reply:
@NitishKumar06 changed it back to the previous reference text